app/controllers/concerns/active_insights/basic_authentication.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ActiveInsights::BasicAuthentication
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :authenticate_by_http_basic
+  end
+
+  def authenticate_by_http_basic
+    return unless http_basic_authentication_enabled?
```

Comment:
Do we need this extra check? If the creds are set i think we are safe to assume auth is enabled.

Reply:
That's what MissionControl is doing, not sure what the benefit is...maybe with ENV vars always being strings you could accidentally set user/pass to an empty string? So having to also set `enabled` makes it explicit that you really do want to turn it on...